http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -510,8 +513,14 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         // general configs
         raw"fullnameOverride=$appChartPrefix-${release.asString}",
         raw"instrumentationEnabled=${config.coaAppConfig.instrumentationEnabled}"
-      ).mkString(",")
-    )
+      )
+
+    val updatedLs = sourceWorkspaceId match { // TODO remove after WDS chart migration
+      case Some(value) => ls ::: List(raw"provenance.sourceWorkspaceId=${value.value}")
```

Comment:
Trying to track this down, but where does `provenance` live in our Helm chart values exactly?

Reply:
ah jk -- found it in one of your other PRs... 🤦